resources/authgear/templates/en/web/authflowv2/setup_oob_otp.html

```diff
@@ -1,70 +1,69 @@
-{{- if eq $.OOBAuthenticatorType "oob_otp_sms" }}
-	{{- if eq .Channel "whatsapp" }}
-		{{ template "setup-oob-otp-title--whatsapp" }}
-	{{ else }}
-		{{ template "setup-oob-otp-title--sms" }}
-	{{- end }}
-{{- end }}
-{{- if eq $.OOBAuthenticatorType "oob_otp_email" }}
-	{{ template "setup-oob-otp-title--email" }}
-{{ end }}
-</h1>
+	{{ $display_otp_input_error := false }}
+	{{ if $.Error }}
+		{{ if eq $.Error.reason "ValidationFailed" }}
+			{{ $display_otp_input_error = true }}
+		{{ end }}
+	{{ end }}
 
-<form class="twc-container-vertical" method="post" novalidate>
-{{ $.CSRFField }}
+	{{ if $.Error }}
+    {{ if not $display_otp_input_error }}
+      <div class="mb-8 alert-message alert-message--error">
+        <span class="alert-message__icon material-icons material-icons--filled">
+          error
+        </span>
+        <span class="alert-message__text">
+          {{ template "authflowv2/__error.html" . }}
+        </span>
+      </div>
+    {{ end }}
+  {{ end }}
 
-{{- if eq $.OOBAuthenticatorType "oob_otp_sms" }}
-<input
-	data-intl-tel-input="true"
-	data-intl-tel-input-class-value="flex flex-row"
-	class="primary-txt text-input w-full"
-	tabindex="0"
-	{{ if $.ShouldFocusInput }}autofocus{{ end }}
-	type="text"
-	inputmode="tel"
-	autocomplete="tel-national"
-	autocapitalize="none"
-	name="x_target"
-	placeholder="{{ template "phone-number-placeholder" }}"
-	data-controller="intl-tel-input"
-	data-action="input->intl-tel-input#input countrychange->intl-tel-input#input"
->
-{{- end }}
+  <div class="space-y-4">
+    {{ template "authflowv2/__header.html" . }}
+    <h1 class="screen-title">
+      {{ template "v2-setup-oob-otp-page-title" }}
+    </h1>
+    <h2 class="screen-description">
+      {{ template "v2-setup-oob-otp-page-subtitle" . }}
+    </h2>
+  </div>
 
-{{- if eq $.OOBAuthenticatorType "oob_otp_email" }}
-<input
-	tabindex="0"
-	{{ if $.ShouldFocusInput }}autofocus{{ end }}
-	class="text-input primary-txt"
-	type="email"
-	name="x_target"
-	autocomplete="email"
-	autocapitalize="none"
-	placeholder="{{ template "email-placeholder" }}"
->
-{{- end }}
+  <form class="mt-8" method="post" novalidate>
+		{{ $.CSRFField }}
 
-<button
-	class="btn primary-btn mt-5"
-	type="submit"
-	name="x_action"
-	value=""
-	data-authgear-event="authgear.button.setup_oob_otp"
->
-{{ template "next-button-label" }}
-</button>
+		{{ $otp_error_message := "" }}
+    {{ if $.Error  }}
+      {{ if $display_otp_input_error }}
+        {{ $otp_error_message = include "authflowv2/__error.html" .  }}
+      {{ end }}
+    {{ end }}
 
-</form>
+		{{- if eq $.OOBAuthenticatorType "oob_otp_sms" }}
+		{{ template "authflowv2/__phone_input.html"
+			(dict
+				"Placeholder" (include "v2-setup-oob-otp-page-phone-number-placeholder" nil)
+				"InputName" "x_target"
+				"ErrorMessage" $otp_error_message
+				"EmptyMessage" (include "v2-setup-oob-otp-page-phone-number-empty-message" nil)
+			)
+		}}
```

Comment:
Sorry I've modified the interface of phone input in the previous pr. Maybe need some modifications here.